crypto/hash/htr/hashtree.go

```diff
@@ -0,0 +1,17 @@
+package htr
+
+import (
+	"github.com/prysmaticlabs/gohashtree"
+)
+
+// VectorizedSha256 takes a list of roots and hashes them using CPU
+// specific vector instructions. Depending on host machine's specific
+// hardware configuration, using this routine can lead to a significant
+// performance improvement compared to the default method of hashing
+// lists.
+func VectorizedSha256(inputList [][32]byte, outputList [][32]byte) {
```

Comment:
For a future PR it would be good to get rid of this and just call the hasher, there's a stack allocation here, error handling and an extra call that we should avoid, specially when hashing 2 chunks.